docs/topics/generators.md

```diff
@@ -115,7 +115,27 @@ chats = await pipeline.run_many(50)
 
 ## Local Models
 
-We have experimental support for both [`vLLM`](https://docs.vllm.ai/en/latest/) 
+Rigging supports local models via the [`ollama`](https://ollama.ai/) provider:
```

Comment:
Provided we essentially have 3 different "local providers" (given that ollama is technically just an API server as well), we might want to establish a short summary of the options and segment the technical details to their own sections.

Reply:
Just to make sure, we're talking LiteLLM, vLLM and Transformers right? The documentation mentions "experimental support" for vLLM, in which way is it less stable than the LiteLLM integration? I'll need some help about these sections as I'm not entirely sure of the support for each in Rigging and the main differences of using one vs another.